<launch>
  <!-- Launch file for velodyne LiDAR -->

    <arg name="rviz" default="true" />

    <rosparam command="load" file="$(find lidar_imu_init)/config/robosense.yaml" />

    <node pkg="lidar_imu_init" type="li_init" name="laserMapping" output="screen"/>

    <param name="point_filter_num" type="int" value="3"/>
    <param name="max_iteration" type="int" value="5" />
    <param name="cube_side_length" type="double" value="2000" />

    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_imu_init)/rviz_cfg/spinning.rviz" />
    </group>
   launch-prefix="gdb -ex run --args" 
</launch>
